From 3fe6e32dd86f1f73d7d273cef540f87b3d958961 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Mon, 06 Jun 2022 08:24:30 +0300
Subject: [PATCH] [SW-89967] gaudi3: remove rom recovery api

It is decided to not support secured firmware update as part of rom
recovery, this patch removes rom recovery api definition from recovery
routine.

Change-Id: Ib4bba3a09533e807a0c5e3e6badb68f6d7cbdf95
---

diff --git a/arch/gaudi3/arc/recovery.c b/arch/gaudi3/arc/recovery.c
index 3b0373c..532ee7e 100644
--- a/arch/gaudi3/arc/recovery.c
+++ b/arch/gaudi3/arc/recovery.c
@@ -117,15 +117,6 @@
 	ext_rom_load_erom_image(HL_FW_EROM_RECOVERY);
 }
 
-static void rom_recovery(void)
-{
-	/* ROM recovery not supported in slave die */
-	if (arcboot_rom_global.die_id)
-		return;
-
-	/* TODO: implement rom recovery */
-}
-
 static void rom_pcie_recovery(void)
 {
 	uint32_t pcie_ctrl_val = arcboot_rom_global.config_fuse & EFUSE_SYSCFG_PCIE_CONTROL_MASK;
@@ -199,7 +190,6 @@
 
 	rom_pcie_recovery();
 	rom_d2d_link_init();
-	rom_recovery();
 }
 
 void execute_recovery(void)
